lib/mayaUsd/nodes/proxyShapeBase.cpp

```diff
@@ -787,7 +787,14 @@ MStatus MayaUsdProxyShapeBase::computeInStageDataCached(MDataBlock& dataBlock)
                 if (nullptr == rootLayer)
                     rootLayer = SdfLayer::FindOrOpen(fileString);
 
-                if (rootLayer) {
+                if (nullptr == rootLayer) {
```

Comment:
This used to be in the else below, but by placing it here it can do the same extra processing existing stages do. There was no reason to exclude new stage from that processing. (Like targeting the session layer if requested.)